cypress/features/regression/create.feature

```diff
@@ -4,6 +4,18 @@ Feature: Create component
   Background: Open Content component page
     Given I open "Create" component page
 
+  @positive
+  Scenario: Verify inner content and colors of Create component
+    # When I open "Create" component page
```

Comment:
add a comment why you commented the step

Reply:
updated

Reply:
updated